### ISSUE TITLE
On multiple screens, new windows are on the wrong screen

### ISSUE THREAD

**Issue by jaredly** [nylas/N1#58]
I have N1 open on my secondary screen, but when I click "new message" or "manage account" or the "feedback" button, they all appear on my primary screen.
This is rather annoying.

Thanks for an awesome product! I'm excited to use it once the sync has finished ;)

**Comment by bengotow** [comment]
Hey @jaredly thanks for downloading N1. This is definitely something I've been feeling as well. It looks like we're specifying a width and height for new windows, but not positioning them. (Though the feedback one we actually are trying to position above the button—that's interesting...) Will check it out!

**Comment by Sunno** [comment]
I actually have that issue with single screen it appears in top left postition, half of the window is hidden and I cannot move it

![image](https://cloud.githubusercontent.com/assets/1751426/10683011/c7b9ab8a-7906-11e5-8edc-4cead996d7e3.png)

**Comment by grinich** [comment]
@Sunno that top-right issue should be fixed as of today's update

**Comment by alex-bender** [comment]
The same situations as @jaredly has.

**Comment by mbilker** [comment]
Does this issue still occur?

**Comment by alex-bender** [comment]
I can not reproduce. "new message"  window appears left aligned on the horizontal center of the screen.

**Comment by mbilker** [comment]
@alex-bender Do you have multiple monitors? If so, when you say left aligned on the horizontal center, do you mean between the multiple screens?

**Comment by alex-bender** [comment]
Yes, I have multiple monitors. And yes, it is between the screens. But I just realized that couple weeks ago I have swap prime and secondary monitors. So for now "new message" window appears on primary screen, doesn't matter where is the main window of Nyla's situated. Tomorrow I will install latest version and try again.

**Comment by mbilker** [comment]
@alex-bender What OS are you again?

**Comment by alex-bender** [comment]
Firstly OS:
`$ uname -a
Linux bender 3.19.0-15-generic #15-Ubuntu SMP Thu Apr 16 23:32:37 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux`
After some time of testing here are what I got:
Main Nylas window opening at its last position. "New message" window -- at the screen where application launcher is. (Gnome 2)
I have downloaded deb package from [download page](https://nylas.com/N1).
Previous version removed with `dpkg -r nylas`
Nylas version from menu is 0.3.32-b11131d

**Comment by mbilker** [comment]
@alex-bender Hey, the window will open in its last state because of `~/.nylas/main-window-state.json` which contains something like:
```
{"version":1,"windowDimensions":{"x":1282,"y":16,"width":1916,"height":1044,"maximized":false},"packageStates":{"attachments":{},"composer":{},"composer-signature":{},"composer-spellcheck":{},"events":{},"message-list":{},"notification-mailto":{},"notifications":{},"preferences":{},"sidebar-fullcontact":{},"tooltip":{},"undo-redo":{},"message-autoload-images":{}}}
```

**Comment by alex-bender** [comment]
I haven't ~/.nylas/main-window-state.json file.

**Comment by jackiehluo** [comment]
From #2839: https://github.com/electron/electron/blob/master/docs/api/screen.md#screengetprimarydisplay

**Comment by jackiehluo** [comment]
Another request: https://github.com/nylas/N1/issues/2999

**Comment by jackiehluo** [comment]
Another request: https://github.com/nylas/N1/issues/3002
